restaurant/views.py

```diff
@@ -186,40 +185,7 @@ def get_restaurants_list(request, page):
 
 
 def get_landing_page(request, page=1):
-    neighbourhoods_filter = request.GET.getlist("neighbourhood")
-    categories_filter = request.GET.getlist("category")
-    price_filter = request.GET.getlist("price")
-    rating_filter = None
-    compliant_filter = None
-    sort_option = None
-    favorite_filter = None
-    user = None
-    if request.user.is_authenticated:
-        user = request.user
-    if request.GET.getlist("ratingfrom"):
-        rating_filter = float(request.GET.getlist("ratingfrom")[0])
-    keyword = request.GET.get("search")
-
-    restaurant_list = get_restaurant_list(
-        page,
-        12,
-        keyword,
-        neighbourhoods_filter,
-        categories_filter,
-        price_filter,
-        rating_filter,
-        compliant_filter,
-        sort_option,
-        favorite_filter,
-        user,
-    )
-
-    parameter_dict = {
-        "restaurant_list": json.dumps(restaurant_list, cls=DjangoJSONEncoder),
-        "page": page,
-        "keyword": json.dumps({"keyword": keyword}),
-    }
-    return render(request, "browse.html", parameter_dict)
+    return render(request, "browse.html")
```

Comment:
Are we not using the previous function for landing page?

Reply:
yes